built/authorization_service_configuration.d.ts

```diff
@@ -15,15 +16,17 @@ export interface AuthorizationServiceConfigurationJson {
  * More information at https://openid.net/specs/openid-connect-discovery-1_0-17.html
  */
 export declare class AuthorizationServiceConfiguration {
+    oauthFlowType: string;
     authorizationEndpoint: string;
     tokenEndpoint: string;
     revocationEndpoint: string;
     endSessionEndpoint?: string | undefined;
     userInfoEndpoint?: string | undefined;
-    constructor(authorizationEndpoint: string, tokenEndpoint: string, revocationEndpoint: string, // for Revoking Access Tokens
+    constructor(oauthFlowType: string, authorizationEndpoint: string, tokenEndpoint: string, revocationEndpoint: string, // for Revoking Access Tokens
     endSessionEndpoint?: string | undefined, // for OpenID session management
     userInfoEndpoint?: string | undefined);
     toJson(): {
+        oauthFlowType: string;
```

Comment:
Maintain the attribute consistency eg: oath_flow_type

Reply:
done